### ISSUE TITLE
Clarify license

### ISSUE THREAD

**Issue by greuff** [Bill-Gray/sat_code#3]
On your web page https://www.projectpluto.com/sat_code.htm you say the code is in the public domain, whereas in the repository here (and in the "lunar" repo) there is a LICENSE file that contains the GPL license. Please clarify which license applies.

**Comment by Bill-Gray** [comment]
You're right.  Thanks for pointing this out.
The original artsat code,  from Neoklis Kyriazis,  was in the public domain,  and I had intended that that license would be preserved.  A few years ago,  I released a lot of code under the GPL and went around making sure it all mentioned the licensing.  It appears I put that license here,  where it didn't belong.
Just to be clear : the 'lunar' repo _is_ GPLed.  The artsat code is under the MIT license (which is admittedly one step from public domain,  but it's a pretty short step.)  Hope this code is useful to you!
